Leaderboard page

florisdobber opened the pull request:
Created a leader board page based on the Figma designs. Also added a picture to the assets as part of the new page. See screenshots below for result

![leaderboard](https://user-images.githubusercontent.com/44882599/80138499-2da35b80-85a5-11ea-803c-2ed41c62d700.jpg)
![progress](https://user-images.githubusercontent.com/44882599/80138502-2ed48880-85a5-11ea-9c00-c1048be0bcac.jpg)

jackblanc commented on src/views/Leaderboard.vue:
```diff
@@ -0,0 +1,187 @@
+      <!-- My progress -->
+      <div class="progress" v-if="!showPlayers">
+        <h2>Team 3</h2>
+        <img src="../assets/tree 2.png" alt="Icon">
```
Let's avoid adding images into the assets folder. I'll write a ticket to make an S3 instance for LLB (and SFTT).

florisdobber replied:
Removed from assets folder

jackblanc commented on src/views/Leaderboard.vue:
```diff
@@ -0,0 +1,187 @@
+</script>
+
+<style scoped lang="less">
+.container {
```
Don't know how any of this works really, but looks good.

jackblanc commented on src/views/Leaderboard.vue:
```diff
@@ -0,0 +1,187 @@
+    <div class="board">
+      <!-- heading to select view -->
+      <div class="heading">
+        <button class="active" @click="openPlayers" v-if="showPlayers">Players</button>
```
could we toggle the classname here instead of having two tags and the v-if?

jackblanc commented on src/views/Leaderboard.vue:
```diff
@@ -0,0 +1,187 @@
+      </div>
+      <!-- Players -->
+      <div class="rows" v-if="showPlayers">
+        <!-- replace with v-for once data is loaded in -->
```
You should create a separate component for each of the Players and My Progress pages. Generally, we want to always break down the page into the smallest pieces we can.

florisdobber commented:
- moved players and progress views out to components
- implemented a v-for in players based on dummy data
- slightly adjusted spacing on views
- removed previously added file from assets
- toggled class names for simplicity
